components/Blog/blogPage.js

```diff
@@ -18,10 +19,13 @@ function useToRenderBlog () {
 function BlogPage () {
   const [blogs, schools, selectedSchool, updateSelectedSchool] = useToRenderBlog()
   return (
-    <div className='flex'>
-      <SideNav schools={schools} selectedSchool={selectedSchool} onSchoolSelected={updateSelectedSchool} />
-      <BlogList blogs={blogs} selectedSchool={selectedSchool} />
-    </div>
+    <React.Fragment>
+      <Head title='Blog Post' />
```

Comment:
The title on the blog home page should be something like "Blog – The Teacher Fund",  not "Blog Post".